corehq/motech/repeaters/management/commands/migrate_records.py

```diff
@@ -0,0 +1,42 @@
+from inspect import cleandoc
```

Comment:
This command (migrating couch repeater records to SQL) will have no use once migration is complete, right? Would a one-off script make more sense for this? I'd rather not create another command users need to ignore in ./manage.py (and I'd imagine it is a slight performance hit every time ./manage.py is executed)

Reply:
We usually end up checking scripts like this in if only because they also need to be run by third parties that host HQ. We can delete them after we're reasonably confident everyone should have run them.

This comment reminded me, @kaapstorm how will the migration experience be for third parties? Will this command be integrated into a django migration like in the [auto-managed](https://commcare-hq.readthedocs.io/migration_command_pattern.html?highlight=auto%20managed%20migration) pattern? Apologies if you've already discussed this - I'm quite late to the party, haven't been paying close attention to this migration recently.

Reply:
(@kaapstorm ignore this question, I'm going to go read the doc about the repeaters migration plan)